### ISSUE TITLE
Table header tags are not closed

### ISSUE THREAD

**Issue by ArthurClemens** [fireproofsocks/inspecto#3]
This results in the table being enclosed in the header.

Example output:
```
<h6>
  <a href="MySchema.html"
    >MySchema</a
  ><!--6-->
  <ul>
    <li>Source:</li>
    <li>Primary Key(s): []</li>
  </ul>
  <table class="inspecto_schema">
    ...
  </table>
</h6>
```

**Comment by fireproofsocks** [comment]
What's the input that produces this output?

**Comment by fireproofsocks** [comment]
Ah, I see it now.

**Comment by fireproofsocks** [comment]
Resolved with PR #4
